utils/python/nexus_nei2016_linker.py

```diff
@@ -153,30 +153,63 @@ def create_target_name(workdir,fname,month,target_date):
     return struct_name
 
 
+def get_hemco_simulation_time(file_path):
+    from datetime import datetime
+    from datetime import timedelta
+    with open(file_path,'r') as reader:
+        # skip the first three comment lines
+        reader.readline() 
+        reader.readline()
+        reader.readline()
+        # get the start time 
+        start = reader.readline()
+        # get the end time
+        end = reader.readline()
+        start_time = datetime.strptime(start,'START:   %Y-%m-%d %H:00:00\n')
+        end_time = datetime.strptime(end,'END:     %Y-%m-%d %H:00:00\n')
```

Comment:
This block should be implemented as:
```
with open(file_path,'r') as reader:
    read line
    remove comments starting with #
    if line starts with START:
        read start_time
    if line start with END:
        read end_time
```
check that both start_time and end_time are defined upon loop exit, otherwise quit